DEVELOPMENT.md

```diff
@@ -1,10 +1,17 @@
-# Jolly Roger development
+# Jolly Roger development tips
 
-Jolly Roger is written in TypeScript using the Meteor web framework. To run
-Jolly Roger locally, you'll need to first
-[install Meteor](https://www.meteor.com/install).
+Jolly Roger is written in [TypeScript](https://www.typescriptlang.org/) using the
+[Meteor web framework](https://www.meteor.com/) version 2. To run Jolly Roger locally, you'll
+need to first [install Meteor](https://www.meteor.com/install).
 
-## Run the server
+Meteor v2 (and therefore Jolly Roger) requires [Node.js](https://nodejs.org/)
+v14 for both development and production. This is an old and deprecated version
+of Node, so you will probably want to use a version manager such as 
+[nvm](https://github.com/nvm-sh/nvm), [Volta](https://volta.sh/),
+[asdf](https://asdf-vm.com/) or (my favorite) [mise](https://mise.jdx.dev/) so
+you're not stuck with Node v14 on your whole system.
```

Comment:
Not sure whether or not this is worth actually specifying, but you only need to install node yourself if you're running a production build. In development mode, the meteor CLI comes with its own node.

Reply:
Ooh I thought it was just its own `npm`! Weird, when I got the version wrong I was getting build issues...

Reply:
Ah, that's because I was using "npm" instead of "meteor npm"

Reply:
Doc fixed!